README.md

```diff
@@ -63,41 +63,6 @@ To inspect Vdsm service status:
 
 Vdsm logs can be found at `/var/log/vdsm/*.log` (refer to README.logging for further information).
 
-
-## Development environment setup
-
-Fork the project on https://github.com/oVirt/vdsm.
-
-Clone your fork:
-
-    sudo dnf install -y git
-    git@github.com:{user}/vdsm.git
-
-Enable oVirt packages for Fedora:
-
-    sudo dnf copr enable -y nsoffer/ioprocess-preview
-    sudo dnf copr enable -y nsoffer/ovirt-imageio-preview
-
-Install additional packages for Fedora, CentOS, and RHEL:
-
-    sudo dnf install -y `cat automation/check-patch.packages`
-
-Create virtual environment for vdsm:
-
-    python3 -m venv ~/.venv/vdsm
-    source ~/.venv/vdsm/bin/activate
-    pip install --upgrade pip
-    pip install -r docker/requirements.txt
-    deactivate
-
-Before running vdsm tests, activate the environment:
-
-    source ~/.venv/vdsm/bin/activate
-
-When done, you can deactivate the environment:
-
-    deactivate
-
```

Comment:
Removing this to special readme is good, but the new shiny readme should be linked from this file. Maybe add very short section on "Contributing" where we point the read to the new readme for developers.